kirtonBCIlab/boccia-bci
# 146 menu polish

# [BOC-146](https://bci4kids.atlassian.net/browse/BOC-146): Polishing - make the menus look nicer
Functionality should be untouched, but many prefabs were modified, as well as the main scene.
## Changes:
- added DynaPuff font, license, and font asset
- set all textual menu elements to use new font
- adjusted font and button state colours for all menu elements
- adjusted scale and layouts of game menus
## To Test:
- click through menus to check that everything is usable
- ensure changes won't conflict with ongoing tasks
